lib/resources/Account.js

```diff
@@ -1,15 +1,151 @@
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        K._get(K.Account.ACCOUNT_RESOURCE + '/pagination', params, {}, callback);
+    },
+
+    timeline: function (id, params, callback) {
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        K._get(K.Account.ACCOUNT_RESOURCE + '/' + id + '/timeline', params, {}, callback);
+    },
+
+    addCustomFields: function (id, customFields, user, reason, comment, params, callback) {
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        var options = {user: user, reason: reason, comment: comment};
+        K._post(K.Account.ACCOUNT_RESOURCE + '/' + id + '/customFields', customFields, params, options, callback);
+    },
+
+    getCustomFields: function (id, params, callback) {
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        K._get(K.Account.ACCOUNT_RESOURCE + '/' + id + '/customFields', params, {}, callback);
+    },
+
+    removeCustomFields: function (id, customFields, user, reason, comment, params, callback) {
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        var options = {user: user, reason: reason, comment: comment};
+        params.customFieldList = customFields.map(function (item) {
+            return item.customFieldId;
+        }).join(',');
+        K._delete(K.Account.ACCOUNT_RESOURCE + '/' + id + '/customFields', params, options, callback);
+    },
+
+    addTags: function (id, tags, user, reason, comment, params, callback) {
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        var options = {user: user, reason: reason, comment: comment};
+        params.tagList = tags.map(function (item) {
+            return item.tagDefinitionId;
+        }).join(',');
+        K._post(K.Account.ACCOUNT_RESOURCE + '/' + id + '/tags', {}, params, options, callback);
+    },
+
+    getTags: function (id, params, callback) {
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        K._get(K.Account.ACCOUNT_RESOURCE + '/' + id + '/tags', params, {}, callback);
+    },
+
+    removeTags: function (id, tags, user, reason, comment, params, callback) {
+        if (typeof params === 'function') {
+            callback = params;
+            params = {};
+        }
+        var options = {user: user, reason: reason, comment: comment};
+        params.tagList = tags.map(function (item) {
+            return item.tagDefinitionId;
+        }).join(',');
+        K._delete(K.Account.ACCOUNT_RESOURCE + '/' + id + '/tags', params, options, callback);
+    },
+
+    addEmail: function (id, email, user, reason, comment, params, callback) {
```

Comment:
We could simplify the API and have the user pass the string directly, instead of an object (`{email: 'XXX'}`).